### DISCUSSION TITLE
Documentation Testing - Using Multiple Views with View States

### DISCUSSION THREAD

**Discussion by jaballogian** [visgl/deck.gl#6934] [Q&A]
I'm testing this https://deck.gl/docs/developer-guide/views#using-multiple-views-with-view-states documentation by creating this codesandbox https://codesandbox.io/s/learn-deck-gl-s1jmeb?file=/src/pages/ViewsAndProjections/UsingMultipleViewsWithViewState.jsx

What is the expected result of running the code from the documentation above?

Here is what I got
![image](https://user-images.githubusercontent.com/24468466/169499621-fd9b486f-273d-4008-a93a-f7404f80bc61.png)
Here are what I understood from my result above:
1. (orange arrow) the `FirstPersonView` was rendered while the `MapView` was not rendered
2. (green arrow) every time I interacted with either the `FirstPersonView` or outside it, the console always printed "main" (which meant that I interacted with the `MapView`. I never could interact with the `FirstPersonView`

**Comment by Pessimistress** [comment]
The code in the documentation is a concise version of the [minimap example](https://deck.gl/examples/multi-view/). There isn't anything to see unless you render some layers.
